mappings/net/minecraft/world/gen/chunk/SurfaceChunkGenerator.mapping

```diff
@@ -12,7 +12,18 @@ CLASS net/minecraft/class_3754 net/minecraft/world/gen/chunk/SurfaceChunkGenerat
 	FIELD field_16580 noiseSizeX I
 	FIELD field_16581 upperInterpolatedNoise Lnet/minecraft/class_3537;
 	FIELD field_16648 AIR Lnet/minecraft/class_2680;
+	FIELD field_16649 STRUCTURE_SMOOTHING_MAP [F
 	FIELD field_24773 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24774 generatorType Lnet/minecraft/class_5284;
+	FIELD field_24775 DEPTH_SMOOTHING_MAP [F
```

Comment:
iirc this works generally for transitions between biomes and not specifically with the depth? if so, it might be worth naming just like `BIOME_SMOOTHING_MAP`